hack/coverage.sh

```diff
@@ -44,18 +44,20 @@ generate_cover_data() {
 }
 
 show_cover_report() {
-  echo "Generating ${1} report"
-  go tool cover -${1}="$profile" ${2}
+  echo "Generating $1 report"
+  go tool cover -"$1"="$profile" "$2"
 }
 
-generate_cover_data $(go list ./...)
+generate_cover_data "$(go list ./pkg/...)"
```

Comment:
We do have tests in `./cmd/...` as well.

Reply:
👍